server.js

```diff
@@ -1,10 +1,26 @@
 const express = require('express')
 const app = express()
-const port = 3001
-app.get('/ping' , (req, res) => {
-    res.send('Hello everyone')
+const mongoose = require('mongoose')
+const port = 3000
+
+const uri = 'mongodb+srv://Surya:sun@asapcluster.3osvgms.mongodb.net/?retryWrites=true&w=majority';
+
+
+
+app.get('/' ,(req,res) =>{
```

Comment:
Move the database connection to the server startup instead of connecting on every request. This will improve performance and avoid unnecessary connections. [important]